docs/sdk/authentication/payment-delegation-database.md

```diff
@@ -0,0 +1,143 @@
+# Payment Delegation Database <!-- omit in toc -->
+
+The Payment Delegation Database is currently under development, with an work-in-progress relay that'll make using it easier, but is currently deployed on the Lit network, `manzano`.
+
+The Payment Database consists of three contracts:
+
+- [PaymentDelegation.sol](https://github.com/LIT-Protocol/lit-assets/blob/develop/blockchain/contracts/contracts/lit-node/PaymentDelegation.sol)
+  - This is an [ERC-2535 Diamond proxy](https://eips.ethereum.org/EIPS/eip-2535)
+  - Currently deployed at: `0x180BA6Ec983019c578004D91c08897c12d78F516`
+- [PaymentDelegationFacet.sol](https://github.com/LIT-Protocol/lit-assets/blob/develop/blockchain/contracts/contracts/lit-node/PaymentDelegation/PaymentDelegationFacet.sol)
+  - The actual database implementation contract
+  - Currently deployed at: `0x06E81394482FF7429Acc86bdFC695853435FDCD4`
+- [LibPaymentDelegationStorage.sol](https://github.com/LIT-Protocol/lit-assets/blob/develop/blockchain/contracts/contracts/lit-node/PaymentDelegation/LibPaymentDelegationStorage.sol)
+  - The database's proxy storage contract
+
+## Using the Payments Database <!-- omit in toc -->
+
+- [`delegatePayments`](#delegatepayments)
+- [`getPayersAndRestrictions`](#getpayersandrestrictions)
+- [`getPayers`](#getpayers)
+- [`getUsers`](#getusers)
+- [`undelegatePayments`](#undelegatepayments)
+- [`setRestriction`](#setrestriction)
+- [`getRestriction`](#getrestriction)
+- [`delegatePaymentsBatch`](#delegatepaymentsbatch)
+- [`undelegatePaymentsBatch`](#undelegatepaymentsbatch)
+
+In order to start using the Payment Database, you should fetch the address of the latest implementation:
+
+```js
+const paymentDelegationFacet = await ethers.getContractAt(
+    'PaymentDelegationFacet',
+    await paymentDelegationDiamond.getAddress()
```

Comment:
how does the user get this `paymentDelegationDiamond.getAddress()' as this looks to be taken from a test case.

Reply:
https://github.com/LIT-Protocol/networks it's in this repo, or they can use the resolver etc